internal/target/queue/queue_test.go

```diff
@@ -45,11 +45,7 @@ import (
 // See newTestQueueDir to create testing queue from an existing directory.
 // It is called responsibility to remove queue directory created by this function.
 func newTestQueue(t *testing.T, target module.DeliveryTarget) *Queue {
-	dir, err := ioutil.TempDir("", "maddy-tests-queue")
-	if err != nil {
-		t.Fatal("failed to create temporary directory for queue:", err)
-	}
-	return newTestQueueDir(t, target, dir)
+	return newTestQueueDir(t, target, t.TempDir())
```

Comment:
os.RemoveAll call in cleanQueue function below seems redundant now. Can you remove it?

Reply:
Done :heavy_check_mark: